src/Components/comments.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import CommentCard from "./commentCard";
+
+const Comments = ({ comments }) => {
+  return (
```

Comment:
Would be good to provide some kind of message or indication that there if there are no comments 
So the user doesn't think they're just loading or something (article 37 has no comments if you'd like to test this)